.github/workflows/unit-tests.yml

```diff
@@ -117,3 +127,36 @@ jobs:
           name: pytest-warning-report-html
           path: |
             reports/pytest_warnings/warning_report_all.html
+
+  # Combine and upload coverage reports.
+  coverage:
+    needs: run-tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ 3.8 ]
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
```

Comment:
This isn't the latest version of this action. I think the other actions here are also pinned at old versions.